ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -523,6 +523,39 @@ public void setMaxLength(ReactEditText view, @Nullable Integer maxLength) {
     view.setFilters(newFilters);
   }
 
+  @ReactProp(name = "autoComplete")
+  public void setTextContentType(ReactEditText view, @Nullable String autocomplete) {
+    if ("username".equals(autocomplete)) {
```

Comment:
You should check for null first. It is a valid type on the JS side to pass undefined or null. Since you don't check for it and return out it will cause an exception to be thrown. We should allow null and set auto fill to its default.